packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -2202,6 +2202,13 @@ class SizedOverflowBox extends SingleChildRenderObjectWidget {
 /// without making the child available for hit testing, and without taking any
 /// room in the parent.
 ///
+/// Animations continue to run in offstage children and may cause unintentional
+/// battery drain.
```

Comment:
Refer to TickerMode here or under "See also" ?

Reply:
Done.

Reply:
rather than "and may cause unintentional battery drain" I would say `, and therefore use battery and CPU time, regardless of whether the animations end up being visible.`